action_files/test_models/src/models.py

```diff
@@ -17,8 +19,27 @@ def main():
     Y_train_df, Y_test_df, Y_hat_df, Y_fitted_df, S_df, tags = get_data()
 
     reconcilers = [BottomUp(),
+                   BottomUpSparse(),
+                   TopDown(method="average_proportions"),
+                   TopDownSparse(method="average_proportions"),
+                   TopDown(method="proportion_averages"),
+                   TopDownSparse(method="proportion_averages"),
+                   MiddleOut(middle_level="Country/State", top_down_method="average_proportions"),
+                   MiddleOutSparse(middle_level="Country/State", top_down_method="average_proportions"),
+                   MinTrace(method='ols'),
+                   MinTrace(method='wls_struct'),
+                   MinTrace(method='wls_var'),
+                   MinTrace(method='mint_cov'),
                    MinTrace(method='mint_shrink'),
-                   MinTrace(method='ols')]
+                   MinTraceSparse(method='ols'),
+                   MinTraceSparse(method='wls_struct'),
+                   MinTraceSparse(method='wls_var'),
+                   OptimalCombination(method='ols'),
+                   OptimalCombination(method='wls_struct'),
+                   ERM(method='closed'),
+                #    ERM(method='reg'), # This is so insanely slow that we don't run it
+                #    ERM(method='reg_bu'), # This is so insanely slow that we don't run it
```

Comment:
If these methods don't work we may as well remove them, people may be trying them out in a grid search or similar and just wasting time.

Reply:
Agree, I'll be adding a deprecation notice in a future PR (or e.g. an exception when people try to run it with more than 10 timeseries)